Main.java

```diff
@@ -0,0 +1,722 @@
+    }
+    public void setPrenume(String prenume) {
+        this.prenume = prenume;
+    }
+
+    public int getSalariu(){
+        return this.SALARIU_BAZA;
+    }
+
+    Angajat(String nume, String prenume){
+        this.nume = nume;
+        this.prenume = prenume;
+    }
+
+    @Override
+    public String toString() {
+        return "Angajat{" +
+                "nume='" + nume + '\'' +
+                ", prenume='" + prenume + '\'' +
+                ", salariu=" + SALARIU_BAZA +
+                '}';
+    }
+}
+
+class Bodyguard extends Angajat{
+    private String firmaSecuritate;
+
+    Bodyguard(String nume, String prenume, String firmaSecuritate) {
+        super(nume, prenume);
+        this.firmaSecuritate = firmaSecuritate;
+    }
+
+    public int getSalariu() {
+        return this.SALARIU_BAZA + 1500;
+    }
+
+    // Getters and Setters
+
+    public String getFirmaSecuritate() {
+        return firmaSecuritate;
+    }
+    public void setFirmaSecuritate(String firmaSecuritate) {
+        this.firmaSecuritate = firmaSecuritate;
+    }
+
+    @Override
+    public String toString() {
+        return "Bodyguard" + "\n" +
+                "firmaSecuritate: " + firmaSecuritate;
+    }
+}
+
+class Bibliotecar extends Angajat{
+    // grad = I/II/III/debutant
+    private String grad;
+
+    Bibliotecar(String nume, String prenume, String grad) {
+        super(nume, prenume);
+        this.grad = grad;
+    }
+
+    public int getSalariu() {
+        return this.SALARIU_BAZA + 2000;
+    }
+
+    // Getters and Setters
+
+    public String getGrad() {
+        return grad;
+    }
+    public void setGrad(String grad) {
+        this.grad = grad;
+    }
+
+    @Override
+    public String toString() {
+        return "Bibliotecar" + "\n" +
+                "grad: " + grad;
+    }
+}
+
+interface Interfata{
+    void vizualizareAng(Angajat[] ang);
+
+    Angajat[] adaugaAng(Angajat[] ang, Angajat a);
+
+    Angajat[] citireDateAngajatNou(Scanner scanner, Angajat[] ang, ClasaServiciu cs);
+
+    Angajat[] eliminaAng(Angajat[] ang, int idx);
+
+    void vizualizareListe(Client[] clImpr, Client[] clRetur);
+
+    void vizualizeazaAut(Autor[] aut);
+
+    void vizualizeazaCar(Carte[] c);
+
+    void vizualizeazaSect(Sectiune[] s);
+
+    void vizualizeazaCarDupaAut(String nume, Carte[] c);
+
+    void vizualizeazaCarDupaSec(String nume, Sectiune[] s);
+
+    Client[] citireDateClientImpr(Scanner scanner, ClasaServiciu cs, Client[] clImpr, Carte[] c);
+
+    Client[] citireDateClientRetur(Scanner scanner, ClasaServiciu cs, Client[] clRetur, Client[] clImpr, Carte[] c);
+
+    boolean verificaNume(String numeCarte, Carte[] c);
+
+    boolean verificaDisponibilitate(String numeCarte, Carte[] c);
+
+    Carte gasesteCarte(String numeCarte, Carte[] c);
+
+    Client[] adaugaClient(Client[] clArray, Client cl);
+
+    boolean  verificaDate(Client[] clImpr, String cnp, String numeCarte);
+
+}
+
+class ClasaServiciu implements Interfata{

```

Comment:
find a more relevant name for the class - usually we use multiple services, one for each main entity - in your case you can have ServiciuAngajat that can hold all methods for an Angajat, ServiciuCarte, so on, so you can also split this service into multiple ones (Also the interface: AngajatInterface, ServiciuAngajat, CarteInterface, ServiciuCarte, etc